examples/bigcommerce/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    plugins: {
+      tailwindcss: {},
+      autoprefixer: {},
+    }
+  }
```

Comment:
There's no newline at the end of this file and it seems the indentation is out of whack